src/games/arena/repositories/arena/actions/admin/create-or-finish-game.ts

```diff
@@ -69,6 +70,7 @@ export async function endGame(userRequesting: User) {
       return getGameError(arenaCommandReply.noActiveGame());
     }
     await enableAllItems(game._gameTypeId, transaction);
+    await endActiveRound(transaction);
     await game.endGame(transaction);
     await activateAllArenaZones(transaction);
     await publishArenaMessage(arenaCommandReply.channelEndGame(game), true);
```

Comment:
bug fix: games were ending with active rounds. when creating a new game, the active round from the old game was being loaded.